kindlecomicconverter/pdfjpgextract.py

```diff
@@ -25,6 +25,11 @@
 from random import choice
 from string import ascii_uppercase, digits
 
+# skip stray images a few pixels in size in some PDFs
+# typical images are many thousands in length
+# https://github.com/ciromattia/kcc/pull/546
+STRAY_IMAGE_LENGTH_THRESHOLD = 300
```

Comment:
> Ok, I got it now, I changed the boundaries to 287 since some image lengths were being to that number. 

_Originally posted by @Derzz in https://github.com/ciromattia/kcc/issues/546#issuecomment-1599315223_

@axu2 
shouldn't it be lower than 300?

Reply:
I used 300 since I saw 159 160 287 in testing and thought it was a reasonable threshold. Real images are many thousands in length so definitely won't overlap.